stream-chat-android-compose/src/main/res/values-en/strings.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
Too fee string resources to split into different resource files. We can do it later.

Reply:
Also fine - I love that you've split tehm up in regions at least, so it's easier to navigate them in general.

Looks great! :]